sprocs/group_roles_validate.sql

```diff
@@ -0,0 +1,77 @@
+CREATE FUNCTION
+    group_roles_validate (
```

Comment:
Is it necessary for this code to be a sproc? I suspect we could do this instead with a single `SELECT` and some JavaScript, which would be my preference.

Reply:
Moved to a single select statement in `groups.sql`.